### ISSUE TITLE
ReferenceError: lineObj is not defined

### ISSUE THREAD

**Issue by soldiertt** [gkovacs/read-each-line#1]
Hi,

I'm using the module in a red hat openshift cloud. I need to run my application in strict mode.
And i encounter the error :
`ReferenceError: lineObj is not defined`
using your module.

```
ReferenceError: lineObj is not defined
    at readEachLine (/var/lib/openshift/56ea692a2d527155290001b6/app-root/runtime/repo/node_modules/read-each-line/read-each-line.js:104:24)
```

could you check if your module is strict mode compatible, thank you

**Comment by gkovacs** [comment]
think this particular error should be fixed in version 1.0.5 (just published via npm), could you give it a try?

**Comment by soldiertt** [comment]
Indeed, it's fixed, thank you
